### PR TITLE
Optional labelsAccessor and valuesAccessor

### PR DESCRIPTION
**Pull request by jeans11**
Hi guys !

So, it's update for use ```labelsAccessor``` and ```valuesAccessor```.

Just , I don't update the new stacked bar chat. In fact, for this, the label of each bar is the ```x``` propertie of object in values array, right ?!

### CODE REVIEW

**Comment by MatthewHerbst** [comment]
Minor request: could you add the two properties to `src/mixins/CartesianChartPropsMixin.js`? That way any Chart that uses that mixin, you don't need to worry about changing the props. (Ex: `LineChart`, `AreaChart`, `BarChart`, etc.)

**Comment by jeans11** [comment]
Absolutely !!!

File: src/areachart/AreaChart.jsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ module.exports = React.createClass({
       interpolate: false,
       interpolationType: null,
       className: 'rd3-areachart',
-      hoverAnimation: true
+      hoverAnimation: true,
+			labelsAccessor: d => d.name,
```
Comment (yang-wei) [inline src/areachart/AreaChart.jsx]: Please follow the indentation. If you are using Sublime Text you'll need to turn tab into space

File: src/areachart/AreaChart.jsx [hunk 2]
```diff
@@ -34,7 +36,9 @@ module.exports = React.createClass({
 
     var props = this.props;
 
-    var data = props.data;
+    var data           = props.data;
+		var labelsAccessor = props.labelsAccessor;
```
Comment (yang-wei) [inline src/areachart/AreaChart.jsx]: prefer ` var { labelsAccessor, valuesAccessor } = props `

File: src/areachart/AreaChart.jsx (hunk 1 above)
Comment (jeans11) [inline src/areachart/AreaChart.jsx]: Sorry, I use Vim. He changes indentation.

File: src/areachart/AreaChart.jsx (hunk 2 above)
Comment (jeans11) [inline src/areachart/AreaChart.jsx]: Ok.

File: src/linechart/DataSeries.jsx [hunk 3]
```diff
@@ -36,13 +36,15 @@ module.exports = React.createClass({
     var xScale = props.xScale;
     var yScale = props.yScale;
     var xAccessor = props.xAccessor,
-        yAccessor = props.yAccessor;
+        yAccessor = props.yAccessor,
+				valuesAccessor = props.valuesAccessor,
```
Comment (yang-wei) [inline src/linechart/DataSeries.jsx]: please make the var declaration same too =)

**Comment by yang-wei** [comment]
Close due to conflict and inactive. Feel free to reopen for discussion

**Pull request closed without merging**